spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,36 @@
+describe AdminUser do
+  describe '.find_or_create' do
+    let(:admin_user) { AdminUser.find_or_create }
+    it 'returns an instance of AdminUser' do
+      expect(admin_user).to be_an_instance_of(AdminUser)
+    end
+    it 'ensures the returned AdminUser api_key is not null' do
+      expect(admin_user.api_key).not_to be(nil)
+    end
+    context 'when there is no existing AdminUser' do
+      it 'will create a new AdminUser' do
+        expect { AdminUser.find_or_create }.to change { AdminUser.count }.by(1)
+      end
+      it 'will set first_name and last_name to the default values' do
+        expect(admin_user.first_name).to eq('admin')
+        expect(admin_user.last_name).to eq('admin')
+      end
+    end
+    context 'when there is existing AdminUser' do
+      let!(:existing_admin_user) do
+        AdminUser.create(user_name: AdminUser::ADMIN_USERNAME, first_name: 'jerry', last_name: 'foo')
+      end
+      let(:returned_user) { AdminUser.find_or_create }
```

Comment:
It's preferable to reuse the same object as before (`admin_user`). I don't think you need to worry about the variable name being less explicit, since the tests are short and will rely on the English descriptions to convey what they're doing.